site/content/docs/main/tilt.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Rapid iterative Velero development with Tilt "
+layout: docs
+---
+
+## Overview
+This document describes how to use [Tilt](https://tilt.dev) with any cluster for a simplified
+workflow that offers easy deployments and rapid iterative builds.
+
+This setup allows for continuing deployment of the Velero server and, if specified, any provider plugin or the restic daemonset.
+It does this work by:
+
+1. Deploying the necessary Kubernetes resources, such as the Velero CRDs and Velero deployment
+2. Building a local binary for Velero and (if specified) provider plugins as a `local_resource`
+3. Invoking `docker_build` to live update any binary into the container/init container and trigger a re-start
+
+Tilt will look for configuration files under `velero/tilt-resources`. Most of the
+files in this directory are gitignored so you may configure your setup according to your needs.
+
+## Prerequisites
+1. [Docker](https://docs.docker.com/install/) v19.03 or newer
+1. A Kubernetes cluster (does not have to be Kind)
+1. [Tilt](https://docs.tilt.dev/install.html) v0.12.0 or newer
+1. Clone the [Velero project](https://github.com/vmware-tanzu/velero) repository
+   locally
+2. Clone any [provider plugin(s)](https://velero.io/plugins/) you want to make changes to and deploy (optional, must be configured to be deployed by the Velero Tilt's setup, [more info below](#provider-plugins))
+
+## Getting started
+
+### tl;dr
```

Comment:
The tl;dr is missing setting up the `tilt-provider.json` files.

Reply:
The tilt-provider.json for the providers listed in the sample file are already all setup (after everything is merged), nothing to do there.

Reply:
Doh, I missed those. Thanks!